Loop/Public/SwiftUI/EnvironmentValues.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
```

Comment:
shouldn't it be under `#canImport(SwiftUI)`

Reply:
Only if we need to support building with Xcode 10 and earlier.